src/ast.rs

```diff
@@ -12,55 +12,55 @@ impl<'a> fmt::Display for Name<'a> {
 
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub enum BinOp {
-    Plus,
-    Minus,
-    Times,
-    Divide,
-    Modulo,
+    Add,
+    Sub,
+    Mul,
+    Div,
+    Rem,
     And,
     Or,
     BitAnd,
     BitOr,
     BitXor,
     Shr,
     Shl,
-    Equal,
-    NotEqual,
-    LessThan,
-    GreaterThan,
-    LessThanOrEqual,
-    GreaterThanOrEqual,
+    Eq,
+    Neq,
+    Lt,
+    Gt,
+    Lte,
+    Gte,
 }
 
 impl<'a> fmt::Display for BinOp {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         use BinOp::*;
         match self {
-            Plus => write!(f, "+"),
-            Minus => write!(f, "-"),
-            Times => write!(f, "*"),
-            Divide => write!(f, "/"),
-            Modulo => write!(f, "%"),
+            Add => write!(f, "+"),
+            Sub => write!(f, "-"),
+            Mul => write!(f, "*"),
+            Div=> write!(f, "/"),
```

Comment:
there's an space missing here. Can you apply `cargo fmt` on an individual commit to format the code just in case?